documentation.lisp

```diff
@@ -1064,7 +1288,118 @@ See STATUS")
     "Returns non nil if this converation has been marked not red")
 
   (function last-status
-    "Returns the last STATUS of the conversation"))
+    "Returns the last STATUS of the conversation")
+
+  (type featured-tag
+    "Representation of tags most visited by a profile
+
+See ID
+See NAME
+See STATUSES-COUNT
+See LAST-STATUS-AT")
+
+  (function last-status-at
+    "Timestamp of the last status mentioning the tag")
+
+  (type filter
+    "Representation of a filter (defined by the user) to make invisible unwanted statuses
+
+See PHRASE
+See FILTER-CONTEXT
+See EXPIRES-AT
+See IRREVERSIBLE
+See WHOLE-WORD")
+
+  (function phrase
+    "Returns the filter's text")
+
+  (function filter-context
+    "Returns Where the filter is applied.
+
+Possible values are:
+
+\"home\"          --- home timeline
+\"notifications\" --- notifications
+\"public\"        --- public timeline
+\"thread\"        --- message's thread")
```

Comment:
Since this is a limited set of possible values, it would be better to translate to keywords rather than strings.